awx_collection/test/awx/test_workflow_job_template_node.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import pytest
+
+from awx.main.models import WorkflowJobTemplateNode, WorkflowJobTemplate, JobTemplate
+
+
+@pytest.mark.django_db
+def test_create_workflow_job_template_node(run_module, admin_user, organization, project, inventory):
+    wfjt = WorkflowJobTemplate.objects.create(organization=organization, name='foo-workflow')
+    WorkflowJobTemplate.objects.create(organization=None, name='foo-workflow')  # to test org scoping
+
+    jt = JobTemplate.objects.create(
+        project=project,
+        inventory=inventory,
+        playbook='helloworld.yml',
+        name='foo-jt'
+    )
+
+    result = run_module('tower_workflow_job_template_node', {
+        'identifier': '42',
```

Comment:
splash in some unicode

Reply:
I'm not 100% sure the Ansible sanity tests will allow me, but if not, I should be able to add a similar test to our awx functional tests.